<?xml version="1.0" encoding="UTF-8"?>
<!-- Launch the swarm control system control for the EZ-RASSOR. -->

<launch>
  <arg name="robot_count" default="3"/>
  <arg name="lander_coords" default="2 0"/>
  <arg name="digsite_x_coords" default="5 -5 8"/>
  <arg name="digsite_y_coords" default="7 6 -5"/>
  <arg name="world" default="nasa_moon_dem"/>
  <arg name="elevation_map" default="nasa_moon_dem.jpg"/>

  <!-- Launch the autonomous control node. -->
  <node pkg="ezrassor_swarm_control"
        name="swarm_control"
        type="swarm_control"
        output="screen">
    <param name="robot_count" value="$(arg robot_count)"/>
    <param name="digsite_x_coords" value="$(arg digsite_x_coords)"/>
    <param name="digsite_y_coords" value="$(arg digsite_y_coords)"/>
    <param name="lander_coords" value="$(arg lander_coords)"/>
    <param name="world" value="$(arg world)"/>
    <param name="elevation_map" value="$(arg elevation_map)"/>
  </node>
</launch>
